view/frontend/layout/default.xml

```diff
@@ -24,6 +24,14 @@
             <block class="Bolt\Boltpay\Block\Js" name="boltcss"   template="Bolt_Boltpay::css/boltcss.phtml"/>
             <block class="Bolt\Boltpay\Block\Js" name="replacejs" template="Bolt_Boltpay::js/replacejs.phtml"/>
         </referenceBlock>
+        <!-- for blank theme-->
+        <referenceBlock name="top.links">
+            <block class="Bolt\Boltpay\Block\Js" name="boltaccount_blank" template="Bolt_Boltpay::boltaccount.phtml" before="-"/>
+        </referenceBlock>
+        <!-- for luna theme-->
+        <referenceBlock name="header.links">
+            <block class="Bolt\Boltpay\Block\Js" name="boltaccount_luna" template="Bolt_Boltpay::boltaccount.phtml" before="-"/>
+        </referenceBlock>
```

Comment:
Will this work every time, for every theme? Is there a better way to uniformly insert the button? Theme agnostic one.
Perhaps something similar to how we are injecting button in the minicart, JS + CSS selectors. It is not elegant, but it works. On the other hand, we may want to move over from that sort of implementation. Not sure. Just an idea to discuss. Even better, to do it Magento way, perhaps we can conditionally / programmatically set the referenceBlock and inject our block in it. cc @matt-thomason @njfaries @johnmccombs1

Reply:
@vitaliyreznikov After taking a second look I think you should probably leave it like this. You add it to both top.links and header.links, and one of the blocks should exist in (most of) all the themes. If it does not we could deal with it in the integration when we get there. If there is a case with both blocks present and two Bolt account buttons added we could hide one with CSS until we find a better solution.

Reply:
Thank you Daniel, it's important thoughts. 

It's very logical for other themes developers save this block names "header.links" and "top.links". But of course, we can't be sure. This project is in early beta. I propose to leave code as it is and looks at how it works. If we handle with a theme that has another block name, we can just add it here.

Injection button via JS has two disadvantages:
- we need to add one more field into our settings, but we already have a lot
- we need to inject a button only after M2 reformat menu to the mobile view and we don't have any events to know when it happens. Probably we can find solution but right now I don't have it.

Long story short, now we support two themes and probably we are good with other themes. We can go ahead and maybe create a new task to investigate how we can handle with different themes.

Reply:
A task in M2P to investigate other themes is a good idea @vitaliyreznikov . Please title it is `[Spike]` at the beginning of the title.

Reply:
@matt-thomason Task is created https://boltpay.atlassian.net/browse/M2P-33